includes/view/AngelTypes_view.php

```diff
@@ -569,6 +569,10 @@ function AngelType_view_info(
     $admin_angeltypes,
     $supporter
 ) {
+    $required_info_show = !auth()->user()->userAngelTypes()->pluck('angel_types.id')->contains($angeltype->id)
```

Comment:
```suggestion
    $required_info_show = !auth()->user()
        ->userAngelTypes()
        ->where('angel_types.id', $angeltype->id)
        ->count()
```
would fit a bit better but its fine as it is